### PR TITLE
Update quickstart-sql-database.md

### PR DESCRIPTION
**Pull request by ruimaiaMSFT**
Replaced "but Windows Authentication is supported. " by "but for Azure SQL Database AAD Universal MFA authentication is also supported", as Windows Authentication is not supported on Azure SQL Database.

### CODE REVIEW

**Comment by Jak-MS** [comment]
@erinstellato-ms 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.

Note, before this can be merged, the disallowed acronym 'AAD' must be changed to 'Azure AD' 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/azure-data-studio/quickstart-sql-database.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Use Azure Data Studio to establish a connection to your Azure SQL Database serve
    
    ![Screenshot showing the Welcome Azure Delta Studio dialog box with the Next connection option called out.](media/quickstart-sql-database/new-connection-icon.png)
```
Comment (erinstellato-ms) [inline docs/azure-data-studio/quickstart-sql-database.md]: Can you please change "Delta" to "Data" in the image description?  (I know that wasn't your original fix, but I noticed it while in here).  Thanks!

**Review by erinstellato-ms — CHANGES_REQUESTED** [review]
Please update description

**Comment by Court72** [comment]
@ruimaiaMSFT - Can you respond to the requested changes, or let us know if you would like us to commit them? 

@erinstellato-ms - After the changes are made, please add a sign-off command to signal that the PR is ready to merge.

**Comment by ruimaiaMSFT** [comment]
> @ruimaiaMSFT - Can you respond to the requested changes, or let us know if you would like us to commit them?
> 
> @erinstellato-ms - After the changes are made, please add a sign-off command to signal that the PR is ready to merge.

Changes done.

**Comment by erinstellato-ms** [comment]
#sign-off

**Comment by erinstellato-ms** [comment]
#sign-off

File: docs/azure-data-studio/quickstart-sql-database.md [hunk 2]
```diff
@@ -34,9 +34,9 @@ Use Azure Data Studio to establish a connection to your Azure SQL Database serve
 
 1. The first time you run Azure Data Studio the **Welcome** page should open. If you don't see the **Welcome** page, select **Help** > **Welcome**. Select **New Connection** to open the **Connection** pane:
    
-   ![Screenshot showing the Welcome Azure Delta Studio dialog box with the Next connection option called out.](media/quickstart-sql-database/new-connection-icon.png)
+   ![Screenshot showing the Welcome Azure Data Studio dialog box with the Next connection option called out.](media/quickstart-sql-database/new-connection-icon.png)
 
-2. This article uses SQL sign-in, but also supports Windows authentication. Fill in the following fields using the server name, user name, and password for your Azure SQL server:
+2. This article uses SQL sign-in, but for Azure SQL Database Azure AD Universal MFA authentication is also supported. Fill in the following fields using the server name, user name, and password for your Azure SQL server:
```
Comment (rwestMSFT) [inline docs/azure-data-studio/quickstart-sql-database.md]:
```suggestion
2. This article uses SQL sign-in, but for Azure SQL Database, Azure AD Universal MFA authentication is also supported. Fill in the following fields using the server name, user name, and password for your Azure SQL server:
```

**Pull request merged**